_layouts/base.html

```diff
@@ -21,6 +21,19 @@
     <![endif]-->
     <link href="{{BASE_PATH}}/atom.xml" type="application/atom+xml" rel="alternate" title="Inbucket News Feed">
     <link href="{{BASE_PATH}}/images/favicon.png" rel="shortcut icon" type="image/png">
+    
+    <script language="javascript">
+      // Enforce SSL
+      (function (root) {
+        "use strict";
+        var h = root ? root.location.hostname : "",
+        p = root ? root.location.protocol : "";
+        if ("http:" === p && !(/^(localhost|127.0.0.1)/).test(h)) {
```

Comment:
Could you add a test for 192.168.* (and maybe 10.*) to this?  As it is implemented currently it would actually break my workflow; I develop on Windows over SSH+tmux+vim to a linux host; my browser ends up pointed to 192.168.1.x.